ITR/data/base_providers.py

```diff
@@ -358,44 +392,63 @@ def get_company_intensity_and_production_at_base_year(self, company_ids: List[st
                           self.column_config.GHG_SCOPE12,
                           self.column_config.GHG_SCOPE3]]
         ei_at_base = self._get_company_intensity_at_year(base_year, company_ids).rename(self.column_config.BASE_EI)
-        return company_info.merge(ei_at_base, left_index=True, right_index=True)
+        df = company_info.merge(ei_at_base, left_index=True, right_index=True)
+        df.reset_index('scope', inplace=True)
+        cols = df.columns.tolist()        
+        df = df[cols[1:3] + [cols[0]] + cols[3:]]
```

Comment:
why these exact columns?
what are columns "1:3", "0" and "3:" ?
shouldn't we access columns by names?

Reply:
We don't know the names of 3: (the year columns vary based on user input).  This little dance is just reordering the 'scope' column we popped out to come after sector and region, which is how it's usually presented.  Pandas is quite tolerant of column order generally (when names are used, which mostly they are), so this is just for the user's benefit.  we could certainly change this to `df = df[['sector', 'region', 'scope'] + cols[3:]]`.

Reply:
user's benefit is exactly what i care about.
yes, let's do for user's benefit